game/resource/English/ability/items/tooltip_moriahs_ring.txt

```diff
@@ -0,0 +1,14 @@
+//==================================================================
+// Moriah's Ring
+//==================================================================
+
+"DOTA_Tooltip_Ability_item_moriahs_ring"                          "Moriah's Ring"
+"DOTA_Tooltip_Ability_item_recipe_moriahs_ring"                   "Moriah's Ring Recipe"
+"DOTA_Tooltip_Ability_item_moriahs_ring_Description"              "<h1>Toggle: Sangromancy</h1> When active Moriah's Ring grants %sangromancy_spell_amp%%% spell amplification but inflicts %sangromancy_self_damage%%% of the damage you deal with spells and items also to yourself"
```

Comment:
"but you take %sangromancy_self_damage%%% you deal with spells and items" is probably better phrasing.

Reply:
"but you take %sangromancy_self_damage%%% of the damage you deal with spells and items", no?

Reply:
Ah yes. Missing a phrase there. :p